### DISCUSSION TITLE
Directional Light, Shadow Type 설정 질문

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/XRStudy#55] [7. Q&A]
안녕하세요
Percetion 을 돌려 보는 중인데 초기 프로젝트 설정에서 Directional Light에서 Shadow Type을 No Shadow로 설정을 하라고 하는데, 
Inspector에서 보이지가 않네요. 
에디터 버전 차이 때문이지, 혹시 Shadow를 없애는 것이 뭔지 아시나요? 
그림 Shadow 영역에 Shadow Map Enable을 해제하라는 것인지... 
혹시라도 설정 관련해서 아시는게 있다면 답변 부탁드립니다. 

![Screenshot 2024-07-16 at 16 10 22](https://github.com/user-attachments/assets/c01a8c76-0b66-4d43-861d-ed66a1966ee9)

감사합니다.

**Comment by jongfeel** [comment]
이게 HDRP 세팅이라서 그렇습니다.
Shadow Map에 Enable 체크 풀면 no shadow가 됩니다.
